modules/core/src/main/scala/effect_zoo/contests/cdown/Kyo.scala

```diff
@@ -2,18 +2,20 @@ package effect_zoo.contests.cdown
 import effect_zoo.contests.{Cdown, Contender}
 import kyo.*
 
-
 object Kyo extends Cdown.Entry(Contender.Kyo):
-  def program: Int < Vars[Int] =
-    Vars.get[Int].flatMap: n =>
-      if n <= 0
-      then n
-      else Vars.set(n - 1).andThen(program)
+  def program: Int < Var[Int] =
+    Var
+      .get[Int]
+      .flatMap: n =>
+        if n <= 0
+        then n
+        else Var.set(n - 1).andThen(program)
```

Comment:
More idiomatic for performance-sensitive code:

```suggestion
    Var.use[Int]: n =>
        if n <= 0
        then n
        else Var.setAndThen(n - 1)(program)
```